tests/integration/docker/info.bats

```diff
@@ -31,6 +31,6 @@ setup() {
 @test "Container info" {
 	container=$(random_name)
 	$DOCKER_EXE run -itd --name $container busybox
-	$DOCKER_EXE info| grep "Containers: 1"
+	$DOCKER_EXE info| grep "^ Running: 1"
```

Comment:
what about using a regex instead of a fixed number, so in case we have other containers running, the test does not fail.